### PR TITLE
refactor: replace lodash calls with native

### PR DESCRIPTION
**Pull request by jrassa**

### CODE REVIEW

File: src/app/core/admin/messages/list-messages.component.ts [hunk 1]
```diff
@@ -51,10 +50,8 @@ export class ListMessagesComponent extends AbstractPageableDataComponent<Message
 	ngOnInit() {
 		this.alertService.clearAllAlerts();
 		this.route.params.subscribe((params: Params) => {
-			if (
-				toString(params[`clearCachedFilter`]) === 'true' ||
-				null == this.messageService.cache.listMessages
-			) {
+			const clearCachedFilter = params?.[`clearCachedFilter`] ?? '';
```
Comment (tzfx) [inline src/app/core/admin/messages/list-messages.component.ts:53]: TIL '??' is a thing.

File: src/app/core/page-title.service.ts [hunk 2]
```diff
@@ -5,6 +5,7 @@ import { ActivatedRoute, NavigationEnd, Router } from '@angular/router';
 import capitalize from 'lodash/capitalize';
 import isEmpty from 'lodash/isEmpty';
 import { filter, map, switchMap, tap } from 'rxjs/operators';
+import { StringUtils } from '../common/string-utils.service';
```
Comment (tzfx) [inline src/app/core/page-title.service.ts]: Is this unused?

**Comment by tzfx** [comment]
Take that, lodash!

File: src/app/core/page-title.service.ts (hunk 2 above)
Comment (jrassa) [reply src/app/core/page-title.service.ts]: Removed

**Pull request merged**